source/_integrations/tag.markdown

```diff
@@ -42,6 +42,30 @@ Home Assistant has a dedicated panel that allows you to manage your tags. You ca
 
 ![Tag user interface in Home Assistant](/images/blog/2020-09-15-home-assistant-tags/tag-ui.gif)
 
+
+## Using Scanned Tags as a Trigger
+
+To use a tag scan as the trigger for an automation use the ```tag``` platform:
+
+{% raw %}
```

Comment:
Why is this escaped?

Reply:
I appreciate your time addressing my first attempt at adding to the docs, but I don't know what "escaped" means. I was just following the Jukebox example when I added the {% raw %}

Reply:
It can be removed, there is no conflict with liquid tags in your code fenced block, so we don't need to tell Jekyll this is raw data.